### PR TITLE
Add example use case

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3214

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -15,9 +15,13 @@ rootProject.name = 'ballerina-persist'
 
 include ':checkstyle'
 include ':persist-ballerina'
+include ':persist-examples'
+
```
Comment (kalaiyarasiganeshalingam) [inline settings.gradle:19]: Shall we remove the extra new line?
Please check all the places

File: examples/redcross/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[package]
+org = "wso2"
+name = "persist_redcross"
```
Comment (daneshk) [inline examples/redcross/Ballerina.toml:3]: Shall we give a different name? like `basic_entity`

Comment (kaneeldias) [reply examples/redcross/Ballerina.toml]: Fixed in [f7c186b](https://github.com/ballerina-platform/module-ballerina-persist/pull/5/commits/f7c186b06ba8d5a2f9183a613d63d2cec1f1bdf5)

File: settings.gradle (hunk 1 above)
Comment (kaneeldias) [reply settings.gradle:19]: Fixed in [f7c186b](https://github.com/ballerina-platform/module-ballerina-persist/pull/5/commits/f7c186b06ba8d5a2f9183a613d63d2cec1f1bdf5)

**Review by daneshk — APPROVED** [review]

File: examples/build.gradle [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+apply plugin: 'java'
+
+description = 'Ballerina - Persist Examples'
+
+def ballerinaDist = "${project.rootDir}/target/ballerina-runtime"
+def examples = ["redcross"]
```
Comment (daneshk) [inline examples/build.gradle:26]:
```suggestion
def examples = ["medical_center"]
```

Comment (kaneeldias) [reply examples/build.gradle]: Fixed in [0b1c605](https://github.com/ballerina-platform/module-ballerina-persist/pull/5/commits/0b1c6053251a714640f3a3dd7f0879915715dbc3)

**Review by daneshk — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**